src/core/data-tests.js

```diff
@@ -24,34 +24,37 @@ const lang = defaultLang in l10n ? defaultLang : "en";
 
 function toListItem(href) {
   
-  // const modifiedHref = href.split("/").pop().split(".")[0];
+  var [testFile] = new URL(href).pathname.split("/").reverse();
```

Comment:
nit: rename this to something else, don't reassign it below (otherwise it's confusing to know what this actually is)